src/calendar/view/eventeditor/AttendeeListEditor.ts

```diff
@@ -0,0 +1,277 @@
+import m, { Children, Component, Vnode } from "mithril"
+import { MailRecipientsTextField } from "../../../gui/MailRecipientsTextField.js"
+import { RecipientType } from "../../../api/common/recipients/Recipient.js"
+import { ToggleButton } from "../../../gui/base/buttons/ToggleButton.js"
+import { Icons } from "../../../gui/base/icons/Icons.js"
+import { ButtonSize } from "../../../gui/base/ButtonSize.js"
+import { Checkbox } from "../../../gui/base/Checkbox.js"
+import { lang } from "../../../misc/LanguageViewModel.js"
+import { CalendarAttendeeStatus, NewPaidPlans } from "../../../api/common/TutanotaConstants.js"
+import { Autocomplete, TextField, TextFieldType } from "../../../gui/base/TextField.js"
+import { CompletenessIndicator } from "../../../gui/CompletenessIndicator.js"
+import { RecipientsSearchModel } from "../../../misc/RecipientsSearchModel.js"
+import { noOp } from "@tutao/tutanota-utils"
+import { Guest } from "../../date/CalendarInvites.js"
+import { createAttendingItems, iconForAttendeeStatus } from "../../date/CalendarUtils.js"
+import { Icon } from "../../../gui/base/Icon.js"
+import { theme } from "../../../gui/theme.js"
+import { IconButton } from "../../../gui/base/IconButton.js"
+import { BootIcons } from "../../../gui/base/icons/BootIcons.js"
+import { px, size } from "../../../gui/size.js"
+import { createDropdown } from "../../../gui/base/Dropdown.js"
+import { CalendarEventWhoModel } from "../../date/eventeditor/CalendarEventWhoModel.js"
+import { LoginController } from "../../../api/main/LoginController.js"
+import { CalendarEventModel } from "../../date/eventeditor/CalendarEventModel.js"
+import { DropDownSelector } from "../../../gui/base/DropDownSelector.js"
+import { showPlanUpgradeRequiredDialog } from "../../../misc/SubscriptionDialogs.js"
+
+export type AttendeeListEditorAttrs = {
+	/** the event that is currently being edited */
+	model: CalendarEventModel
```

Comment:
only for the shouldShowSendInviteNotAvailable call and the shouldSendUpdates checkbox. the former could be injected in the attrs I guess and the latter could be moved to the CalendarEventWhoModel since it's preparing the SendMailModels anyway.

Reply:
I think it's okay to keep it